stdlib/format.mli

```diff
@@ -53,6 +53,148 @@
    from {!formatter_of_out_channel} (on the standard out channels or others).
 *)
 
+(** {1 Format by example}
+
+  A few warmup examples to get an idea of how Format is used.
+
+  We have a list [l] of pairs [(int * bool)], which the toplevel prints for us:
+  {[# let l = List.init 100 (fun n -> n, n mod 2 = 0);;
+  val l : (int * bool) list =
+  [(0, true); (1, false); (2, true); (3, false); (4, true); (5, false);
+   (6, true); (7, false); (8, true); (9, false); (10, true); (11, false);
```

Comment:
Could we use a shorter example? Like to 10 or 15?

Reply:
trying with 20, should be enough